cassandra-scheduler/src/main/java/com/mesosphere/dcos/cassandra/scheduler/resources/TasksResource.java

```diff
@@ -141,8 +145,9 @@ public void replace(@QueryParam("node") final String name)
     @GET
     @Path("connect")
```

Comment:
Ok, there's an API audit epic here: https://mesosphere.atlassian.net/browse/INFINITY-514.  We'll catch inconsistencies there.